build.sh

```diff
@@ -53,6 +53,9 @@ CMAKE_BIN=`which cmake3 || which cmake`
 mkdir -p $BUILDDIR/artifacts/
 mvn ${MAVEN_ARGS} dependency:copy-dependencies -DoutputDirectory=$BUILDDIR/artifacts/
 
+# Create the resource directory where all native libraries will be copied.
+mkdir -p $BUILDDIR/all/
```

Comment:
OPT: surround with braces the variable

Reply:
Not necessary to change it in this PR as it's a big one

Reply:
I've added a follow-up commit: https://github.com/criteo/JVips/pull/44/commits/e052b418153d30cf5c671eb9c91bb8d503cfc5c4

Reply:
Thank you